### DISCUSSION TITLE
Is 416GiB really enough?

### DISCUSSION THREAD

**Discussion by lrlbh** [Chia-Network/bladebit#60] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/47717790/128606106-0ac110f1-bcf3-48df-a398-2018cb3a88cb.png)
Why does Swp peak at 100GiB?  
Before running the program, my memory usage was less than 1GiB
How much more memory do I need to use it? (32G * 14 or 32 * 16 or ... )

**Comment by harold-b** [comment, marked as answer]
416 is the requirement plus a bit more (megabytes) as stated in the Readme file. No idea about swap behavior as this wasn't design for or tested with swap files.

**Comment by lrlbh** [comment]
416GiB，32 * 14 Is enough！
The front should be SWP caused by high occupancy！
